[Running] python -u "/home/ubuntu/mini_pupper_bsp/Python_Module/MangDang/mini_pupper/display_only.py"
pygame 2.5.2 (SDL 2.28.3, Python 3.10.12)
Hello from the pygame community. https://www.pygame.org/contribute.html
Simple mixer control 'Headphone',0
  Capabilities: pvolume pvolume-joined pswitch pswitch-joined
  Playback channels: Mono
  Limits: Playback -10239 - 400
  Mono: Playback 400 [100%] [4.00dB] [on]
/usr/local/lib/python3.10/dist-packages/pygame/sysfont.py:223: UserWarning: 'fc-list' is missing, system fonts cannot be loaded on your platform
  warnings.warn(
Traceback (most recent call last):
  File "/home/ubuntu/mini_pupper_bsp/Python_Module/MangDang/mini_pupper/display_only.py", line 106, in <module>
    disp.show_image('moonphases/moon11.png')
  File "/home/ubuntu/mini_pupper_bsp/Python_Module/MangDang/mini_pupper/display_only.py", line 63, in show_image
    image = Image.open(image_path)
  File "/usr/local/lib/python3.10/dist-packages/PIL/Image.py", line 3218, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'moonphases/moon11.png'

[Done] exited with code=1 in 5.654 seconds

[Running] python -u "/home/ubuntu/mini_pupper_bsp/Python_Module/MangDang/mini_pupper/display_only.py"
pygame 2.5.2 (SDL 2.28.3, Python 3.10.12)
Hello from the pygame community. https://www.pygame.org/contribute.html
Simple mixer control 'Headphone',0
  Capabilities: pvolume pvolume-joined pswitch pswitch-joined
  Playback channels: Mono
  Limits: Playback -10239 - 400
  Mono: Playback 400 [100%] [4.00dB] [on]
/usr/local/lib/python3.10/dist-packages/pygame/sysfont.py:223: UserWarning: 'fc-list' is missing, system fonts cannot be loaded on your platform
  warnings.warn(
Traceback (most recent call last):
  File "/home/ubuntu/mini_pupper_bsp/Python_Module/MangDang/mini_pupper/display_only.py", line 106, in <module>
    disp.show_image('moonphases/moon11.png')
  File "/home/ubuntu/mini_pupper_bsp/Python_Module/MangDang/mini_pupper/display_only.py", line 63, in show_image
    image = Image.open(image_path)
  File "/usr/local/lib/python3.10/dist-packages/PIL/Image.py", line 3218, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'moonphases/moon11.png'

[Done] exited with code=1 in 6.73 seconds

